core/src/main/scala/org/apache/spark/rdd/util/PeriodicRDDCheckpointer.scala

```diff
@@ -73,8 +76,6 @@ import org.apache.spark.util.PeriodicCheckpointer
  *
  * @param checkpointInterval  RDDs will be checkpointed at this interval
  * @tparam T  RDD element type
- *
- * TODO: Move this out of MLlib?
  */
 private[spark] class PeriodicRDDCheckpointer[T](
```

Comment:
The scaladoc for this class needs to be updated to include this new behaviour. Particularly, the 'WARNINGS' section.

Reply:
@sujithjay, thanks a lot for noticing! 
Just updated the docs a little bit to clarify the new behaviour.